Documentation/Settings.md

```diff
@@ -115,27 +115,33 @@ When the device powers up, should it enter into a special mode. These settings s
 
 On device help text:
 
-O=off | S=soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved
+O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby at no heat heat until moved
```

Comment:
Please leave this as is, since in the past I was asked to reduce the length.
```suggestion
O=off | S=soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved
```